### PR TITLE
Config changes for  5.4 kernel

### PR DESCRIPTION
**Pull request by amshinde**

### CODE REVIEW

File: kernel/build-kernel.sh [hunk 1]
```diff
@@ -201,10 +201,8 @@ get_kernel_frag_path() {
 	results=$( ${cmdpath} -r -n ${all_configs} )
 	# Only consider results highlighting "not in final"
 	results=$(grep "${not_in_string}" <<< "$results")
-	# Do not care about options that are in whitelist if using experimental kernel
-	if [[ ${experimental_kernel} == "true" ]]; then
-		results=$(grep -v -f ${default_config_whitelist} <<< "$results")
-	fi
+	# Do not care about options that are in whitelist
+	results=$(grep -v -f ${default_config_whitelist} <<< "$results")
```
Comment (devimc) [inline kernel/build-kernel.sh]: could you move `fragments/common/experimental/virtio-fs.conf` into `fragments/common/`?

Comment (amshinde) [reply kernel/build-kernel.sh]: @devimc Are the virtio-fs bits merged in stable?
cc @egernst

Comment (devimc) [reply kernel/build-kernel.sh]: nop, I didn't see this, ignore my comment

https://github.com/kata-containers/runtime/blob/1fa10fea78e88a520652463189248d91145518f2/versions.yaml#L160-L163

**Comment by amshinde** [comment]
> We certainly don't need it for 5.4, but I'm concerned that we're going to get in a situation where folks want to support multiple kernels on the same branch of Kata (ie, 4.19 and 5.4 in latest relaese of kata, for example).

So what do you suggest, have a separate whitelist per kernel version?

**Comment by chavafg** [comment]
is this ready? should we remove the DNM label? 
Needed to unblock CI. https://github.com/kata-containers/runtime/pull/2422 is already merged

**Comment by jcvenegas** [comment]
@amshinde @chavafg actually the packages are broken there needs to be updated, @amshinde because the CI in general is merged, can be merged and fix it in a separate PR

**Comment by chavafg** [comment]
ok, thanks @jcvenegas 
+1 to unblock the CI

**Pull request merged**